addons/quicktime/CfgFunctions.hpp

```diff
@@ -0,0 +1,10 @@
+class CfgFunctions {
+    class CBA {
+        class QuickTimeEvent {
+            PATHTO_FNC(generateQTESequence);
+            PATHTO_FNC(getFormattedQTESequence);
+            PATHTO_FNC(runQTE);
+            PATHTO_FNC(keyPressedQTE);
```

Comment:
Order them alphabetically.
```suggestion
            PATHTO_FNC(keyPressedQTE);
            PATHTO_FNC(runQTE);
```